### ISSUE TITLE
System Monitor CPU always 0%

### ISSUE THREAD

**Issue by Y0ngg4n** [linuxdeepin/developer-center#1861]
## Describe the bug
System Monitor shows always 0% usage.

## To Reproduce
Go to System Monitor and check CPU Usage

## Expected behavior
Show correct CPU Usage

## Screenshots
![Screen Capture_deepin-system-monitor_20200817124239](https://user-images.githubusercontent.com/30302975/90387881-3d303e00-e087-11ea-9280-8df4bca450d7.png)

![Screen Capture_deepin-system-monitor_20200817124251](https://user-images.githubusercontent.com/30302975/90387948-5df89380-e087-11ea-8741-6aeba7bb8155.png)

## Enviroments:
```
cat /etc/os-release 
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
BUILD_ID=rolling
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="32;2;24;144;200"
HOME_URL="https://manjaro.org/"
DOCUMENTATION_URL="https://wiki.manjaro.org/"
SUPPORT_URL="https://manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux
```
```
pacman -Q | grep deepin
deepin-account-faces 1.0.11-1.1
deepin-album 5.6.9.13-2
deepin-anything 5.0.1-3
deepin-anything-dkms 5.0.1-3
deepin-api 5.3.0.3-1
deepin-boot-maker 5.4.8-1
deepin-calculator 5.5.27-1
deepin-calendar 5.7.0.5-1
deepin-clone 5.0.3-1
deepin-control-center 5.3.0.6-1
deepin-daemon 5.11.0.21-1
deepin-desktop-base 3:2019.07.10-1
deepin-desktop-schemas 5.8.0.4-1
deepin-dock 5.1.0.11.20200622-1
deepin-draw 5.8.0.19-2
deepin-editor 5.6.26-1
deepin-file-manager 1:5.1.1.25-4
deepin-gtk-theme 2020.06.10-1
deepin-icon-theme 2020.07.24-1
deepin-iconthemes-manjaro 20170918-1
deepin-image-viewer 5.6.3.2-3
deepin-kwin 5.1.0.4-2
deepin-launcher 5.3.0.3-1
deepin-menu 5.0.1-2
deepin-movie 1:5.7.6.29-1
deepin-music 6.0.1.8-2
deepin-network-utils 5.1.0.2-1
deepin-picker 5.0.6-1
deepin-polkit-agent 5.2.0.7-1
deepin-polkit-agent-ext-gnomekeyring 0.1.0-2
deepin-qt-dbus-factory 5.3.0.2-1
deepin-qt5integration 5.1.0.3-1
deepin-qt5platform-plugins 5.0.13-1
deepin-reader 5.7.0.13-1
deepin-screen-recorder 5.8.0.11-3
deepin-screensaver 5.0.3-1
deepin-screensaver-pp 0.0.4-2
deepin-screenshot 5.0.0-4
deepin-session-shell 5.3.0.3-2
deepin-session-ui 5.2.0.16-1
deepin-shortcut-viewer 5.0.2-1
deepin-sound-theme 15.10.6-1
deepin-system-monitor 5.6.12-1
deepin-terminal 5.2.20-1
deepin-turbo 0.0.3-2
deepin-wallpapers 2:1.7.7-1
plymouth-theme-manjaro-deepin-circle 1.0-4
```

**Comment by xmuli** [comment]
I also reproduced

**Comment by juteman** [comment]
This is a confirmed bug.
Thanks for your reports.

**Comment by z966768** [comment]
deepin-system-monitor use the proc file system ,  it seems like the system didn't mount the proc,please use the command like `ps` ，`top` ；`iostate` 。if it is normal, print `ls /proc` and show the screenshot

**Comment by xmuli** [comment]
**Arch + DDE:**
![image](https://user-images.githubusercontent.com/33887845/90994525-8f271580-e5eb-11ea-9d37-526742a6493e.png)
![image](https://user-images.githubusercontent.com/33887845/90994547-a1a14f00-e5eb-11ea-96de-be2d6262e5fb.png)
![image](https://user-images.githubusercontent.com/33887845/90994566-b251c500-e5eb-11ea-9884-74488170440f.png)

**Comment by Y0ngg4n** [comment]
@z966768 the output of ls /proc:

```
ls /proc
1     12     13     13442  161    18     206   263   33    55   828  872         config.gz      net
10    12729  13022  13447  162    180    2061  264   34    56   83   875         consoles       pagetypeinfo
100   12730  13023  13448  163    181    207   265   347   560  832  88          cpuinfo        partitions
101   12741  13045  13449  164    182    208   266   35    57   833  89          crypto         pressure
1012  12744  13069  13459  165    183    209   267   351   58   835  9           devices        sched_debug
1016  12750  13116  13499  166    18393  21    268   36    59   836  90          diskstats      schedstat
1017  12752  13117  14     16606  184    210   269   363   6    837  91          dma            scsi
1018  12757  13124  14963  16625  18469  211   27    37    60   839  910         driver         self
1019  12800  13127  14968  16668  18480  212   277   38    61   84   92          dynamic_debug  slabinfo
102   12802  13130  15     167    185    213   278   380   62   841  93          execdomains    softirqs
1021  12809  13133  15008  16792  186    214   279   382   63   842  94          fb             stat
1022  12814  13142  15009  168    187    215   28    385   64   843  944         filesystems    swaps
1023  12820  13152  15026  16811  188    216   281   39    642  844  946         fs             sys
1025  12831  13155  15034  169    189    22    29    398   65   85   948         interrupts     sysrq-trigger
1026  12848  13166  15036  16907  19     225   3     4     66   850  95          iomem          sysvipc
1027  12849  13189  15037  16944  190    23    30    40    67   853  950         ioports        thread-self
1028  12853  13208  15071  17     191    230   302   41    68   855  951         irq            timer_list
1029  12860  13212  15267  17015  192    231   303   42    69   856  952         kallsyms       tty
103   12862  13218  15292  17051  193    232   304   426   7    857  953         kcore          uptime
1030  12880  13222  153    17062  194    233   306   43    70   858  954         keys           version
104   12886  13246  15320  17075  195    234   3085  44    71   859  955         key-users      vmallocinfo
105   12896  13248  15321  17099  196    235   31    45    72   86   958         kmsg           vmstat
106   12900  13255  15324  171    197    236   3103  46    73   860  96          kpagecgroup    zoneinfo
1065  12919  13259  15390  17123  198    237   315   47    74   861  97          kpagecount
1067  12931  13262  154    17179  199    238   316   48    75   862  98          kpageflags
1069  12949  13297  15402  172    2      239   317   49    76   863  99          latency_stats
107   12952  13302  155    173    20     24    3190  5     77   864  acpi        loadavg
108   12970  13341  156    174    200    240   3199  50    78   865  asound      locks
109   12984  13345  157    175    201    241   32    51    79   866  bootconfig  meminfo
1096  12989  13410  158    176    202    242   3200  52    8    867  buddyinfo   misc
11    12993  13411  159    177    203    2452  324   53    80   868  bus         modules
110   12998  13434  16     178    204    25    326   5330  81   869  cgroups     mounts
117   12999  13435  160    179    205    26    328   54    82   87   cmdline     mtrr
```

**Comment by KimHono** [comment]
Same for me, on 5.8.10-arch1-1, and deepin-system-monitor 5.6.22-3.

**Comment by z966768** [comment]
Thanks for your feedback,we'll fix this soon

**Comment by xmuli** [comment]
> Thanks for your feedback,we'll fix this soon

I have located the specific place, I will fix it later

**Comment by xmuli** [comment]
@Y0ngg4n @kekexi123 
This bug has been fixed in `tag: 5.8.0.1`, see [bc6eadfe47](https://github.com/linuxdeepin/deepin-system-monitor/commit/bc6eadfe47ece57aa19928045db96007ccb1875e)  for details.

And it offers a separate approach to fixing the problem:

```cpp
diff --git a/src/compact_cpu_monitor.cpp b/src/compact_cpu_monitor.cpp
index 96ac2e4..9c1575b 100644
--- a/src/compact_cpu_monitor.cpp
+++ b/src/compact_cpu_monitor.cpp
@@ -85,7 +85,7 @@ CompactCpuMonitor::CompactCpuMonitor(QWidget *parent)
     auto *smo = SystemMonitor::instance();
     Q_ASSERT(smo != nullptr);
     connect(smo->jobInstance(), &StatsCollector::cpuStatInfoUpdated,
-            this, &CompactCpuMonitor::updateStatus);
+            this, &CompactCpuMonitor::updateStatus, Qt::QueuedConnection);
 
     changeFont(DApplication::font());
     connect(dynamic_cast<QGuiApplication *>(DApplication::instance()), &DApplication::fontChanged,
```

**Comment by cestcela** [comment]
Same bug on the brown new Mint 20.3
